demos/TicTacToe/readme.md

```diff
@@ -0,0 +1,69 @@
+Firebase Cocos2d-x TicTacToe Sample
+==========================
+
+Desktop cocos2d-x sample for the Firebase C++ SDK.
```

Comment:
It would be worth noting that this example aims to demonstrate usage of Firebase Auth and Firebase Database.